docker-entrypoint.sh

```diff
@@ -19,8 +19,12 @@ for arg in "$@" ; do
   fi
 done
 
-java -jar tool/target/rdapct-1.0.4.jar -c tool/bin/rdapct_config.json --use-local-datasets "$@" 1>&2
+RCT_VERSION="$(xq -x //rdap-conformance.version pom.xml)"
```

Comment:
As an FYI, this version.properties file will also have the version number in it. But this works too. :)